README.md

```diff
@@ -113,6 +113,17 @@ It takes in a computed application object and returns an augmented score. The id
 module.exports = ({ rating }) => rating;
 ```
 
+It also makes use of a selection script stored in `src/js/hc-scripts/choose-applicants.js`
+This takes a sorted list of applications, a result count limit and an inviteType
+and returns a list of the suggested applications to invite/reject.
+E.g. to just takes the top n in the sorted list:
+
+```
+module.exports = ( (sortedApplications, n, inviteType) => {
+  return sortedApplications.slice(0, Math.min(sortedApplications.length, limit));
```

Comment:
I think it would improve the code clarity!

Sometimes MDN doesn't have all the technical details, but you can always try it out with your own array, or check out the actual specs (http://www.ecma-international.org/ecma-262/6.0/#sec-string.prototype.slice):
> 10. If intEnd < 0, let to be max(len + intEnd,0); otherwise let to be min(intEnd, len).